app/src/main/res/layout/fragment_month.xml

```diff
@@ -10,6 +10,7 @@
         android:id="@+id/month_view_wrapper"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_marginLeft="@dimen/smaller_margin"
```

Comment:
![Screenshot_20250114_180138_Calendar_debug](https://github.com/user-attachments/assets/9fff98ae-f2cd-4950-aa19-4b2646068e00)

Doesn't it look just uneven now with week numbers disabled?